sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3949,24 +3949,24 @@ _ZNK2cl4sycl6detail10image_implILi1EE13getSlicePitchEv
 _ZNK2cl4sycl6detail10image_implILi1EE14getChannelTypeEv
 _ZNK2cl4sycl6detail10image_implILi1EE14getElementSizeEv
 _ZNK2cl4sycl6detail10image_implILi1EE15getChannelOrderEv
+_ZNK2cl4sycl6detail10image_implILi1EE4sizeEv
 _ZNK2cl4sycl6detail10image_implILi1EE7getTypeEv
-_ZNK2cl4sycl6detail10image_implILi1EE9get_countEv
 _ZNK2cl4sycl6detail10image_implILi1EE9get_rangeEv
```

Comment:
I'm not sure it was a good suggestion to remove `image_impl::get_count`.
I think we should either keep it or bump library version to avoid incompatible changes.
Tagging @alexbatashev to clarify.

Reply:
Due to the templated nature of `image`, all methods of `image_impl` are exported (unlike many other impl classes). We must preserve the `get_count()` method to avoid breaking changes.

Reply:
Restored get_count() in image_impl.hpp and its symbols in sycl_symbols_linux.dump

Reply:
Note that I did not add SYCL 2020 deprecation warning to get_count() since it is an impl method, per @vladimirlaz's previous suggestion.